packages/block-library/src/image/edit.native.js

```diff
@@ -458,6 +471,16 @@ export class ImageEdit extends React.Component {
 												<Text style={ styles.uploadFailedText }>{ retryMessage }</Text>
 											</View>
 										}
+										{ ! isUploadInProgress && ! isUploadFailed && finalWidth && finalHeight && showMediaEditorButton &&
+											<MediaEdit allowedTypes={ [ MEDIA_TYPE_IMAGE ] }
+												onSelect={ this.onSelectMediaUploadOption }
+												source={ { uri: url } }
+												openReplaceMediaOptions={ openMediaOptions }
+												render={ ( { open, mediaOptions } ) => {
```

Comment:
My personal preference would be to avoid destructuring and manually calling the `editImageComponent` function here. Since the `editImageComponent` function is only used here, I would destructure in that function (line 400):

```
const editImageComponent = ( { openMediaOptions, mediaOptions } ) => (...
```

so that this `render` line could be compacted down to `render={ editImageComponent }`.

With that said, this is firmly in the realm of personal preference, and what you have is perfectly fine, so feel free to leave it as-is. 👍

Reply:
TIL!

Changed in e1133106d76ca04b9710b5f6d4a3aaf59c2a1a73